packages/babel-plugin/src/babel-plugin.ts

```diff
@@ -131,7 +130,7 @@ export default declare<State>((api) => {
 
             // jsxPragmas currently only run on the top-level compiled module,
             // hence we don't interrogate this.importSources.
-            if (jsxSourceMatches && jsxSourceMatches[1] === COMPILED_IMPORT_SOURCE) {
+            if (jsxSourceMatches && this.importSources.includes(jsxSourceMatches[1])) {
```

Comment:
I believe this could have a side-effect, we're saying "transform `@atlaskit/css` to be `@compiled/react`" which if what _I expect_, but is there any scenario where this isn't intended…?  I'm not exactly sure what `options.importSources` intends to do otherwise—like the `ImportDeclaration` walker works, but omits the jsx pragma.  I'll assume this is just extending that logic to include `jsx` (while before it's just `['styled', 'ClassNames', 'css', 'keyframes', 'cssMap']`).